core/src/main/java/de/jplag/reporting/jsonfactory/ComparisonReportWriter.java

```diff
@@ -42,8 +46,12 @@ public ComparisonReportWriter(Function<Submission, String> submissionToIdFunctio
      * yield the same result.
      */
     public Map<String, Map<String, String>> writeComparisonReports(JPlagResult jPlagResult, String path) {
-        int numberOfComparisons = jPlagResult.getOptions().maximumNumberOfComparisons();
-        List<JPlagComparison> comparisons = jPlagResult.getComparisons(numberOfComparisons);
+        int numberOfMaximumComparisons = jPlagResult.getOptions().maximumNumberOfComparisons();
+        int numberOfAllComparisons = jPlagResult.getAllComparisons().size();
+        int missingComparisons = numberOfAllComparisons - numberOfMaximumComparisons;
+        logger.warn("Total Comparisons: {}. Shown Comparisons: {}. Missing Comparisons: {}.", numberOfAllComparisons, numberOfMaximumComparisons,
```

Comment:
Here would just use logger.info, as this is where the user just configured the CLI to drop some matches.